.gitignore

```diff
@@ -0,0 +1,2 @@
+*/node_modules/**
+*/package-lock.json
```

Comment:
We should put a `.gitingore` in each template directory. That way, a consumer of the template will get the `.gitignore` when they `pulumi policy new`. I'd just copy the ones we have in the other repo, e.g. https://github.com/pulumi/templates/blob/master/aws-typescript/.gitignore